frontend/workflows/projectCatalog/src/details/quick-links.tsx

```diff
@@ -23,22 +23,22 @@ interface QuickLinkProps extends LinkGroupProps {
 }
 
 interface QuickLinkContainerProps {
-  key: string | null | undefined;
+  $key: string | null | undefined;
```

Comment:
why strip this out before rendering instead of just proxying the key value?

Reply:
Not sure, I switched it to a 'keyProp' prop to fix this.